source/Generic/GameRelations/GameRelationsSettingsView.xaml

```diff
@@ -221,7 +221,40 @@
                                 </DockPanel>
                             </Grid>
                         </Expander>
-                        
+
+                        <Expander Margin="0,10,0,0" Header="Advanced">
+                            <StackPanel>
+                                <StackPanel Orientation="Horizontal" Margin="0,10,0,0">
+                                    <TextBlock VerticalAlignment="Center">Jacard similarity requirement per field:</TextBlock>
+                                    <TextBox VerticalAlignment="Center" IsReadOnly="True" MinWidth="35" Margin="10,0"
+                                             Text="{Binding Settings.SimilarGamesControlSettings.JacardSimilarityPerField}" />
+                                    <Slider Minimum="0.4" Maximum="1" IsSnapToTickEnabled="True" TickFrequency="0.01"
+                                            Value="{Binding Settings.SimilarGamesControlSettings.JacardSimilarityPerField}"
+                                            Width="200" VerticalAlignment="Center"/>
+                                </StackPanel>
+
+                                <TextBlock Text="Fields enabled/weights:" Margin="0,10,0,0"/>
```

Comment:
I don't like much the text here. Perhaps it should say "Individual fields settings"